src/App.vue

```diff
@@ -666,6 +669,8 @@ export default {
 			hideTopbar: false,
 			title: '',
 			messages: [],
+			systemDark: false,
+			mq_system_dark: null,
```

Comment:
why don't use a watch for this instead?

Reply:
Also it's not clear to me what `mq_system_dark` is used for, you never re-assign it, I would instead use a let in beforeMount instead

Reply:
First time using Vue. Will re look at the logic here. Thanks for the notes.

Reply:
It's ok no worries I appreciate a lot your help 🙏🏼